content/english/useless-box/learning-concepts.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Learning concepts"
+description: ""
+draft: true
+weight: 2
+---
+## How do we make a "useless" box? Let's learn some basic concepts first!
+
+### What will power the box? Batteries!
+A battery is a chemical pump for electrons(*)! There is a pair of chemical reactions that pump electrons from anode to cathode. 
+
+The battery **voltage** is the potential energy given to electrons as a result of this pump.  
+
+((\*)An electron is a stable subatomic particle with a charge of negative electricity, found in all atoms and acting as the primary carrier of electricity in solids.)
```

Comment:
```suggestion
{{%notice tip%}}
An electron is a stable subatomic particle with a charge of negative electricity, found in all atoms and acting as the primary carrier of electricity in solids.)
{{%/notice%}}
```

Reply:
@ozhang22 can I just include that string you provided? what interprets that text?